migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+include::modules/common-attributes.adoc[]
+:context: migration-from-version-3-to-4-overview
+
+toc::[]
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated. To learn more about migrating from {product-title} 3 to 4 see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[about migrating from OpenShift Container Platform 3 to 4].
```

Comment:
about -> About

(since you're using the "see <title>" method, you use the actual title of the section)

Reply:
acked